### PR TITLE
Third milestone

### PR DESCRIPTION
**Pull request by dannisonarias**
Style the Display component
Style the ButtonPanel component
Style the Button component
Pass “color” and “wide” props to the Button

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -2,14 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, wide, color } = props;
+  const styles = {
```
Comment (whiz25) [inline src/components/Button.js:6]: - [ ] Buttons should take with of 25% except the "0" button. Add a styling for that.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Nice Progress :+1: 
You have come a long way with this PR. Let's make some changes so we can merge.
- [ ] Give your app.js a width of 700px and height of 100%.
- [ ] Add a title to this PR.

### Martin Afani

File: src/components/App.js [hunk 2]
```diff
@@ -2,10 +2,17 @@ import React from 'react';
...
-    <Display />
-    <ButtonPanel />
-  </div>
-);
+const App = () => {
+  const styles = {
+    width: '700px',
+    height: '100%',
+  };
+
```
Comment (shloch) [inline src/components/App.js]: great

**Review by shloch — APPROVED** [review]
## Hello @dannisonarias ,

Great job with the modifications

This is not mandatory, but put app in the center...you can see looks better and symmetrical
calc

### You get a thumbs up...Happy coding

File: src/components/App.js (hunk 2 above)
Comment (dannisonarias) [reply src/components/App.js]: > ## Hello @dannisonarias ,
> Great job with the modifications
> 
> This is not mandatory, but put app in the center...you can see looks better and symmetrical
> calc
> 
> ### You get a thumbs up...Happy coding

thanks! added!

**Pull request merged**